projects/plugins/jetpack/class.jetpack-connection-banner.php

```diff
@@ -251,91 +267,60 @@ public static function enqueue_connect_button_scripts() {
-									'jetpack'
-								);
-								?>
-							</p>
-
-							<p>
-								<?php
-								esc_html_e(
-									'Activate site accelerator tools and watch your page load times decrease—we’ll optimize your images and serve them from our own powerful global network of servers, and speed up your mobile site to reduce bandwidth usage.',
-									'jetpack'
-								);
-								?>
-							</p>
-
-							<div class="jp-banner__button-container">
-								<span class="jp-banner__tos-blurb"><?php jetpack_render_tos_blurb(); ?></span>
-								<a
-										href="<?php echo esc_url( $this->build_connect_url_for_slide( '72' ) ); ?>"
-										class="dops-button is-primary jp-banner__alt-connect-button">
-									<?php esc_html_e( 'Set up Jetpack', 'jetpack' ); ?>
-								</a>
-							</div>
-
-						</div>
-					</div> <!-- end slide 1 -->
+			?>
+			<div class="jp-connection-banner__inner">
+				<div class="jp-connection-banner__content">
+					<div class="jp-connection-banner__logo">
+						<?php echo $jetpack_logo->get_jp_emblem_larger(); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
+					</div>
+					<h2 class="jp-connection-banner__title"><?php esc_html_e( 'Simplify your site security and performance with&nbsp;Jetpack', 'jetpack' ); ?></h2>
+					<div class="jp-connection-banner__columns">
+						<div class="jp-connection-banner__text"><?php esc_html_e( 'Jetpack provides easy-to-use, comprehensive WordPress site security and backups, so you can focus on running your business.', 'jetpack' ); ?></div>
+						<div class="jp-connection-banner__text"><?php esc_html_e( 'Jetpack’s performance features make your site lightning-fast, while also improving your SEO and giving your visitors a better experience.', 'jetpack' ); ?></div>
+					</div>
+					<div class="jp-connection-banner__footer">
+						<div class="jp-connection-banner__text jp-connection-banner__text--caption"><?php jetpack_render_tos_blurb(); ?></div>
+						<a id="jp-connect-button--alt" href="<?php echo esc_url( $this->build_connect_url_for_slide( '72' ) ); ?>" class="jp-banner-cta-button">
+							<?php esc_html_e( 'Set up Jetpack', 'jetpack' ); ?>
+						</a>
+					</div>
+				</div>
+				<div class="jp-connection-banner__image-container">
+					<img class="jp-connection-banner__image-background" src="<?php echo esc_url( plugins_url( 'images/jetpack-connection-banner-background.svg', JETPACK__PLUGIN_FILE ) ); ?>" />
+					<picture>
+						<source
+							type="image/webp"
+							srcset="<?php echo esc_url( plugins_url( 'images/jetpack-connection-image.webp', JETPACK__PLUGIN_FILE ) ); ?> 1x, <?php echo esc_url( plugins_url( 'images/jetpack-connection-image-2x.webp', JETPACK__PLUGIN_FILE ) ); ?> 2x">
+						<img
+							class="jp-connection-banner__image"
+							srcset="<?php echo esc_url( plugins_url( 'images/jetpack-connection-image-2x.png', JETPACK__PLUGIN_FILE ) ); ?> 2x"
```

Comment:
Not 100% sure, but I think you may need to add a 1x version in `srcset` too for users with a browser that might support it, but with a low res screen.